<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MutiWidget</class>
 <widget class="QWidget" name="MutiWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1092</width>
    <height>795</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>501</width>
     <height>551</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>499</width>
      <height>549</height>
     </rect>
    </property>
    <widget class="NewCodeEditor" name="CodeEditor">
     <property name="geometry">
      <rect>
       <x>-10</x>
       <y>0</y>
       <width>511</width>
       <height>551</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QTabWidget" name="detectWidget">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>50</y>
     <width>501</width>
     <height>551</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>欢迎使用代码检测功能</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>0</y>
     <width>431</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pb_format">
      <property name="text">
       <string>规范代码格式</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pb_compile">
      <property name="text">
       <string>编译</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pb_run">
      <property name="text">
       <string>运行</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="detectMenuButton" name="pb_detect">
      <property name="text">
       <string>静态检测</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTabWidget" name="LogAndCmd">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>620</y>
     <width>1021</width>
     <height>171</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Log Information</string>
    </attribute>
    <widget class="QScrollArea" name="scrollArea_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1021</width>
       <height>151</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1019</width>
        <height>149</height>
       </rect>
      </property>
      <widget class="QTextEdit" name="te_log">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1011</width>
         <height>141</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>CMD</string>
    </attribute>
    <widget class="QScrollArea" name="scrollArea_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1021</width>
       <height>161</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1019</width>
        <height>159</height>
       </rect>
      </property>
      <widget class="CmdTextEdit" name="te_cmd">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>-4</y>
         <width>1021</width>
         <height>161</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pb_download">
   <property name="geometry">
    <rect>
     <x>909</x>
     <y>10</y>
     <width>102</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>下载</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_generate">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>10</y>
     <width>102</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>生成审计报告</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>detectMenuButton</class>
   <extends>QPushButton</extends>
   <header location="global">detectmenubutton</header>
  </customwidget>
  <customwidget>
   <class>NewCodeEditor</class>
   <extends>QTextEdit</extends>
   <header location="global">NewCodeEditor</header>
  </customwidget>
  <customwidget>
   <class>CmdTextEdit</class>
   <extends>QTextEdit</extends>
   <header location="global">CmdTextEdit</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
